cli/commands/flags.go

```diff
@@ -707,40 +707,36 @@ func NewVersionFlag(opts *options.TerragruntOptions) cli.Flag {
 // Scaffold/Catalog shared flags
 
 const (
+	LegacyParentConfigName      = "terragrunt.hcl"
```

Comment:
There is same constant in `config.DefaultTerragruntConfigPath`

Reply:
Importing `config.DefaultTerragruntConfigPath` would cause an import loop cycle.

They also have different semantic meaning, even though they have the same value. I'm not referring to the default unit configuration path, "terragrunt.hcl", I'm referring to the legacy root terragrunt configuration name, "terragrunt.hcl".